src/Compilers/CSharp/Portable/Compilation/MemberSemanticModel.NodeMapBuilder.cs

```diff
@@ -255,6 +255,19 @@ public override BoundNode VisitBinaryOperator(BoundBinaryOperator node)
                 throw ExceptionUtilities.Unreachable;
             }
 
+            public override BoundNode VisitDeconstructionAssignmentOperator(BoundDeconstructionAssignmentOperator node)
+            {
+                // For deconstruction declarations, the BoundLocals in the LeftVariables should not be added to the map
+                if (!node.IsDeclaration)
+                {
+                    VisitList(node.LeftVariables);
+                }
+
+                Visit(node.Right);
+                // don't map the deconstruction, conversion or assignment steps
+                return null;
+            }
```

Comment:
>} [](start = 12, length = 1)

It seems like all of our testing for the node map is indirect.  Instead of actually querying to see if the value is in the map, we use an IDE feature that happens to call into the node map.  Is it possible to test this directly?